etc/test.sh

```diff
@@ -18,12 +19,14 @@ if [[ "$1" == "race" ]]; then
 fi
 
 # We run analyzetests on every run, pass or fail. We only run analyzecoverage when all tests passed.
-PION_LOG_WARN=webrtc,datachannel,sctp gotestsum --format standard-verbose $LOGFILE -- -tags=no_skip $RACE $COVER ./...
+PION_LOG_WARN=webrtc,datachannel,sctp gotestsum --format standard-verbose $LOGFILE -- -tags=no_skip $RACE $COVER $TEST_TARGET
 SUCCESS=$?
 
 if [[ $RACE != "" ]]; then
 	cat json.log | go run ./etc/analyzetests/main.go
-	exit $?
+	if [ "$?" != "0" ]; then
```

Comment:
I think we want to `exit $SUCCESS` here? IIUC, this will return whether `analyzetests/main.go` succeeded.

Reply:
good point -- do you want analyze failures to fail the suite?

I don't mind ignoring blips, but would like to know if analyze were failing every time. Ultimately more your area than mine, I'm okay either way.

Reply:
Good point returned. Failing when either tests or analyzetests failed is probably the right way to go.